Dockerfile

```
@@ -1,14 +1,8 @@
-FROM python:3.11.4-buster
-
+FROM python:3.11.4-alpine
 ENV REVIEWDOG_VERSION=v0.17.0
-
-RUN set -eux \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends \
-        git \
-        wget \
+RUN apk --no-cache add git gcc musl-dev \
     && wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s -- -b /usr/local/bin/ ${REVIEWDOG_VERSION} \
-    && pip install detect-secrets[word_list]
+    && pip install detect-secrets[word_list] 
```

Avoid unnecessary changes. This space was added here. 

```suggestion
    && pip install detect-secrets[word_list]
```